src/browserlib/extract-dfns.mjs

```diff
@@ -211,6 +252,17 @@ function definitionMapper(el, idToHeading, usesDfnDataModel) {
     // indicates that definition appears in the main body of the specification)
     definedIn
   };
+
+  // Extract a prose definition in HTML for the term, if available
+  const proseEl = document.querySelector(`[data-defines="#${dfn.id}"]`);
```

Comment:
is there any reason to use `#` in data-defines? unless we expect to allow for other selectors, having the the id would seem sufficient?

Reply:
separately, I wonder if we couldn't handle the "simple" case of `<dt>`/`<dd>` automatically based on a simpler convention (e.g. if `dfn.closest('dt') && dfn.closest('.dfn-desc')` then assumes the `<dd>` is the description of the `<dfn>` in the `<dt>`)? This would match was was being discussed in  https://github.com/w3c/respec/issues/4522 , e.g.
```html
<dl class='dfn-desc'>
  <dt><dfn id='foo'>Foo</dfn></dt>
  <dd>A Foo is a non-sensical word used as a placeholder</dd>
```

(later, we could even have `.dfn-desc` automatically identify the `<dt>` as equivalent to a `<dfn>` - but that probably requires more changes in the authoring tools)

Reply:
> is there any reason to use `#` in data-defines? unless we expect to allow for other selectors, having the id would seem sufficient?

No reason. I started without it, but the current proposal has a `#`...

Reply:
> This would match was was being discussed in [w3c/respec#4522](https://github.com/w3c/respec/issues/4522)

Yes, I expect authoring tools will support that convention... and produce the right `data-defines` attribute in the generated spec so that extraction in Reffy can remain without heuristics!

Reply:
I think it's fine to proceed with the `#` syntax given the reasoning behind it - this would be reasonably easy to simplify later on if needed